codegenerator/cli/templates/static/shared/.cursor/skills/subgraph-migration/references/step-by-step.md

```
@@ -0,0 +1,428 @@
+**Quality Check:**
+```bash
+pnpm codegen && pnpm tsc --noEmit && pnpm test
+```
+
+---
+
+## Step 3: Refactor File Structure
+
+Mirror subgraph file structure using EXACT same filenames:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)